resources/views/account/requestable-assets.blade.php

```diff
@@ -109,8 +109,14 @@ class="table table-striped snipe-table"
 
                                                 </td>
 
+                                                <td>
+                                                    @can('view', \App\Models\AssetModel::class)
+                                                        <a href='{{ url("/models/{$requestableModel->id}") }}' > {{ $requestableModel->name }} </a></td>
```

Comment:
Can we clean up the extra spaces and the extra `</td>` here? We already close the `<td>` outside of the loop.

`<a href='{{ url("/models/{$requestableModel->id}") }}' > {{ $requestableModel->name }} </a></td>`

vs 

`<a href="{{ url("/models/{$requestableModel->id}") }}">{{ $requestableModel->name }}</a>`

Reply:
Sorry, I have it written differently and the tag remained 😬